### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by GreenMan36** [norandiaconu/BeatDownloader#1]
Show an option dialogue, 1 for default path, 2 for custom path.
My Beatsaber is on my D drive.

This should be fairly simple to implement. I'm too lazy though.

Just dump a selector and a variable input to use as a path.

**Comment by norandiaconu** [comment]
Thanks for the suggestion! I added a directory picker that saves your chosen install directory for future runs. Let me know if this works better.

**Comment by GreenMan36** [comment]
Amazing job, you did it better than I thought it could be done in a CMD prompt.

Another few suggestions, lil debug messages.
I was dumb and didn't read the Prerequisites so it would be nice to have:
A message stating oneclick install must be enabled
A message telling you that Xidel is required (I got the generic "xidel is not recognized as an internal or external command" blah blah stuff, a lil echo upon error should do the trick)

And two major improvements in my eyes:
Auto closing the oneclick installer so you won't have https://i.imgur.com/UqRpKNh.png this mess. PS: not sure why it cropped the image but you get the idea..

Also, I changed the delay to 1 and only got a few errors, a second run got them all.
Maybe a delay of 2 is fine? And an automatic second run to catch the outliers? Just be sure to taskkill /pid ModAssistant.exe in between runs.
